### PR TITLE
feat(search): add --full to print complete result text

### LINKED ISSUE #203: search: add --full to print complete message text
## Problem

`slck search messages` (and `search all` / `search files`) truncates each result's text to 60 characters in every output mode. The width is hard-coded (`output.SearchTable(headers, rows, 60)` in `internal/cmd/search/messages.go`, `all.go`, `files.go`). Slack's `search.messages` API returns the full message text, so a search that finds the right message still cannot show what it says.

This matters most when the user token can search but lacks history scopes: `messages read` / `messages thread` then fail with `missing_scope`, and search is the only way to read the message, but only its first 60 characters are visible.

## Proposal

Add a `--full` flag to `slck search messages` (and `search all`, `search files` for consistency) that prints each result's complete text instead of the truncated table column, for example as one block per result with the ref, channel, user and timestamp as a header line. The default table output stays as it is.

## Acceptance

- `slck search messages "<query>" --full` shows each matching message's full text, including multi-line messages.
- Default output is unchanged.
- Tests cover the full-text rendering and the unchanged default.

### PR DESCRIPTION
**Pull request by piekstra**
## Summary

- add `--full` to `search messages`, `search all` and `search files`
- with `--full`, each result prints as a block: the ref, channel, user and time on one line (sanitized the same way as the table), then the complete text indented by two spaces with its line breaks kept; files print their full name and title the same way
- the default one-line table and its 60-character last column are unchanged
- new `output.SearchBlocks` renders the blocks; `renderSearchRows` picks between it and `SearchTable` in the three commands
- README search reference, the `search messages` help examples and the integration test plan list the flag

## Type of Change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update

## Validation

- `make test` passes, including new tests for multi-line full rendering in `search messages`, full text in `search all`, full file names in `search files`, the unchanged default table, and `SearchBlocks` edge cases (sanitized meta cells, CRLF bodies, short rows, no headers)
- `make lint`: 0 issues

## Sanitized empirical results

Built binary, user token, a three-result search in a real workspace:

- with `--full`: one column-header line, three result header lines, 23 indented body lines, the longest 281 characters; line breaks inside messages preserved
- without `--full`: the same results as the existing five-column table, no row longer than 138 characters

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) guide
- [x] My code follows the project's code style
- [x] I have added tests that prove my fix/feature works
- [x] All new and existing tests pass (`make test`)
- [x] I have updated documentation if needed

## Related Issues

Closes #203

### CODE REVIEW

File: internal/output/output.go [hunk 1]
```diff
@@ -144,6 +144,53 @@ func SearchTable(headers []string, rows [][]string, lastColMaxRunes int) {
...
+			meta[i] = sanitizeSearchCell(raw)
+		}
+		if lastIdx > 0 {
+			_, _ = fmt.Fprintln(Writer, strings.Join(meta, " | "))
+		}
+		var body string
+		if lastIdx < len(row) {
+			body = strings.ReplaceAll(row[lastIdx], "\r", "")
+		}
+		for _, line := range strings.Split(body, "\n") {
```
Comment (monit-reviewer) [inline internal/output/output.go]: SearchBlocks writes message/file body text to the terminal with only \r stripped and \n used to split lines; it does not strip ANSI/terminal control sequences (ESC, other C0 controls). Message text is partner/cross-tenant controlled — any workspace member can author a message containing escape sequences (e.g. to move the cursor, clear the screen, rewrite the prompt line, or set a misleading title) that will render when this output is piped to a terminal via --full. This is a lower-severity terminal-injection concern (no code execution), but is new here because --full is the first path that prints raw, untruncated body text instead of a length-capped table cell. sanitizeSearchCell already strips \r/\n for the meta line; consider also stripping/escaping non-printable control characters (e.g. via a small filter that drops runes < 0x20 except intentional ones, or by only ever writing through a helper that escapes ESC) before writing the body in SearchBlocks, matching the treatment already given to meta cells.

*Reply inline to this comment.*

File: internal/output/output.go [hunk 2]
```diff
@@ -144,6 +144,53 @@ func SearchTable(headers []string, rows [][]string, lastColMaxRunes int) {
 	}
 }
 
+// SearchBlocks writes search results with the last column in full, for when
```
Comment (monit-reviewer) [inline internal/output/output.go]: U-G1 / U-I1: `SearchBlocks` is new exported surface on the same agent-parsable search output as `SearchTable`, but its parse contract is only half documented. `SearchTable`'s comment is explicit that a literal `|` becomes U+00A6 so a consumer can split on " | " deterministically. In `SearchBlocks`, only the meta cells go through `sanitizeSearchCell` (line 179); the body at line 186 strips `\r` and is otherwise verbatim, so a body containing `|` stays literal, and the only thing separating a body line from a meta line is the two-space indent. The doc comment states the indent rule but never states that the body is verbatim, so a consumer cannot tell whether `|` in a body line is data or a column separator, and a later refactor could route the body through `sanitizeSearchCell` without any test failing (`TestSearchBlocks` pins `|` escaping on the USER cell only).

Suggested fix: add one line to the doc comment saying the last cell is emitted verbatim apart from carriage returns (no `|` escaping, no newline collapsing), and that the leading two spaces are the marker distinguishing body lines from the meta line; then add a case to `TestSearchBlocks` with a `|` inside the last cell asserting it survives unescaped, so the contract is pinned by a test rather than by prose.

*Reply inline to this comment.*

**Review by monit-reviewer — APPROVED** [review]
## Automated PR Review

**Reviewed commit:** `1dd782878c61`
**Profile:** `claude-monit-reviewer` - **Posting as:** `monit-reviewer`

### Summary

| Reviewer | Findings |
|----------|----------|
| documentation:docs | 0 |
| go:implementation-tests | 0 |
| policies:conventions | 0 |
| architecture:solid-reviewer-agnostic | 1 |
| security:code-auditor | 1 |

<details>
<summary><strong>architecture:solid-reviewer-agnostic (1 finding)</strong></summary>

### Minor - `internal/output/output.go:147`

> U-G1 / U-I1: `SearchBlocks` is new exported surface on the same agent-parsable search output as `SearchTable`, but its parse contract is only half documented. `SearchTable`'s comment is explicit that a literal `|` becomes U+00A6 so a consumer can split on " | " deterministically. In `SearchBlocks`, only the meta cells go through `sanitizeSearchCell` (line 179); the body at line 186 strips `\r` and is otherwise verbatim, so a body containing `|` stays literal, and the only thing separating a body line from a meta line is the two-space indent. The doc comment states the indent rule but never states that the body is verbatim, so a consumer cannot tell whether `|` in a body line is data or a column separator, and a later refactor could route the body through `sanitizeSearchCell` without any test failing (`TestSearchBlocks` pins `|` escaping on the USER cell only).
> 
> Suggested fix: add one line to the doc comment saying the last cell is emitted verbatim apart from carriage returns (no `|` escaping, no newline collapsing), and that the leading two spaces are the marker distinguishing body lines from the meta line; then add a case to `TestSearchBlocks` with a `|` inside the last cell asserting it survives unescaped, so the contract is pinned by a test rather than by prose.

</details>

<details>
<summary><strong>security:code-auditor (1 finding)</strong></summary>

### Minor - `internal/output/output.go:188`

> SearchBlocks writes message/file body text to the terminal with only \r stripped and \n used to split lines; it does not strip ANSI/terminal control sequences (ESC, other C0 controls). Message text is partner/cross-tenant controlled — any workspace member can author a message containing escape sequences (e.g. to move the cursor, clear the screen, rewrite the prompt line, or set a misleading title) that will render when this output is piped to a terminal via --full. This is a lower-severity terminal-injection concern (no code execution), but is new here because --full is the first path that prints raw, untruncated body text instead of a length-capped table cell. sanitizeSearchCell already strips \r/\n for the meta line; consider also stripping/escaping non-printable control characters (e.g. via a small filter that drops runes < 0x20 except intentional ones, or by only ever writing through a helper that escapes ESC) before writing the body in SearchBlocks, matching the treatment already given to meta cells.

</details>

### Reviewer Coverage

- `documentation:docs` — complete (broad); inspected 2 assigned files (9 inspected across reviewers): `README.md`, `integration-tests.md`; skipped: none; constraints: none
- `go:implementation-tests` — complete (broad); inspected 7 assigned files (9 inspected across reviewers): `internal/cmd/search/all.go`, `internal/cmd/search/files.go`, `internal/cmd/search/full_test.go`, `internal/cmd/search/messages.go`, `internal/cmd/search/render.go`, `internal/output/output.go`, `internal/output/output_test.go`; skipped: none; constraints: none
- `policies:conventions` — complete (broad); inspected 5 assigned files (9 inspected across reviewers): `internal/cmd/search/all.go`, `internal/cmd/search/files.go`, `internal/cmd/search/full_test.go`, `internal/cmd/search/messages.go`, `internal/cmd/search/render.go`; skipped: none; constraints: Sibling repos open-cli-collective/cli-common and open-cli-collective/.github were not checked out locally, so shared command-surface and output-and-rendering conventions could only be checked against the repo-local breadcrumbs and summaries, not their full text.
- `architecture:solid-reviewer-agnostic` — complete (broad); inspected 2 assigned files (9 inspected across reviewers): `internal/cmd/search/render.go`, `internal/output/output.go`; skipped: none; constraints: Scope limited to the two assigned files; the three command files, tests and docs were read only as context for the new selection point. Verification run was `go test ./internal/output/... ./internal/cmd/search/...` (both pass); lint was not run.
- `security:code-auditor` — complete (broad); inspected 2 assigned files (9 inspected across reviewers): `internal/output/output.go`, `internal/output/output_test.go`; skipped: none; constraints: none

<details>
<summary>Inspected files (9)</summary>

- `README.md`
- `integration-tests.md`
- `internal/cmd/search/all.go`
- `internal/cmd/search/files.go`
- `internal/cmd/search/full_test.go`
- `internal/cmd/search/messages.go`
- `internal/cmd/search/render.go`
- `internal/output/output.go`
- `internal/output/output_test.go`

</details>

*0 PR discussion threads considered. 0 summarized; 0 resolved.*

---
<details>
<summary>Completed in 3m 07s | ~$3.86 (est.) | claude-sonnet-5, claude-opus-5 | cr 0.10.311</summary>

| Field | Value |
|---|---|
| Model | claude-sonnet-5, claude-opus-5 |
| Reviewers | documentation:docs, go:implementation-tests, policies:conventions, architecture:solid-reviewer-agnostic, security:code-auditor |
| Engine | claude_cli · claude-sonnet-5, claude-opus-5 |
| Reviewed by | cr · monit-reviewer |
| Duration | 3m 07s wall · 5m 40s compute |
| Cost | ~$3.86 (est.) |
| Pricing basis | anthropic-public-2026-09-02 |
| Tokens | 90 in / 18.6k out |

**Per-workstream usage**

- `orchestrator-selection` — claude-sonnet-5
  - In: 6
  - Out: 2.9k
  - Cache read: 105.3k
  - Cache create: 77.0k
  - Cost: ~$0.36 (est.)
  - Duration: 29s
- `documentation:docs` — claude-sonnet-5
  - In: 12
  - Out: 1.6k
  - Cache read: 337.3k
  - Cache create: 79.0k
  - Cost: ~$0.40 (est.)
  - Duration: 34s
- `go:implementation-tests` — claude-sonnet-5
  - In: 16
  - Out: 3.0k
  - Cache read: 511.9k
  - Cache create: 84.7k
  - Cost: ~$0.47 (est.)
  - Duration: 59s
- `policies:conventions` — claude-sonnet-5
  - In: 14
  - Out: 1.9k
  - Cache read: 438.9k
  - Cache create: 91.9k
  - Cost: ~$0.47 (est.)
  - Duration: 45s
- `architecture:solid-reviewer-agnostic` — claude-opus-5
  - In: 26
  - Out: 6.8k
  - Cache read: 841.5k
  - Cache create: 104.3k
  - Cost: ~$1.63 (est.)
  - Duration: 2m 10s
- `security:code-auditor` — claude-sonnet-5
  - In: 10
  - Out: 1.7k
  - Cache read: 267.1k
  - Cache create: 80.3k
  - Cost: ~$0.39 (est.)
  - Duration: 28s
- `orchestrator-rollup` — claude-sonnet-5
  - In: 6
  - Out: 604
  - Cache read: 190.4k
  - Cache create: 21.1k
  - Cost: ~$0.13 (est.)
  - Duration: 12s

</details>

File: internal/output/output.go (hunk 1 above)
Comment (piekstra) [reply internal/output/output.go]: Fixed in 081c7d2: body text goes through stripControl, which drops C0 control characters except newline and tab, DEL, and C1 controls before printing, so escape sequences never reach the terminal. A test covers ESC, BEL and a C1 character.

File: internal/output/output.go (hunk 2 above)
Comment (piekstra) [reply internal/output/output.go]: Fixed in 081c7d2: the doc comment now states the body is verbatim apart from carriage returns and control characters, and that the two-space indent marks a body line, so a literal | in the body is data. TestSearchBlocksBodyVerbatimWithoutControlCharacters pins a | surviving unescaped.

File: internal/output/output.go [hunk 3]
```diff
@@ -144,6 +144,70 @@ func SearchTable(headers []string, rows [][]string, lastColMaxRunes int) {
...
+		if n > 0 || lastIdx > 0 {
+			_, _ = fmt.Fprintln(Writer)
+		}
+		meta := make([]string, lastIdx)
+		for i := 0; i < lastIdx; i++ {
+			var raw string
+			if i < len(row) {
+				raw = row[i]
+			}
+			meta[i] = sanitizeSearchCell(raw)
```
Comment (monit-reviewer) [inline internal/output/output.go]: U-L1: the control-character hardening added in this revision covers only the body, so the property `stripControl`'s comment states ("text another workspace member wrote cannot carry terminal escape sequences into the output") does not hold for the block's own meta line. The meta cells at line 181 go through `sanitizeSearchCell`, which drops `\r`, folds `\n` to a space and escapes `|`, but passes ESC, BEL and C1 characters straight to the terminal. In `search messages` the meta cells include `m.Channel.Name` and `m.Username`, both attacker-influenced display strings, so an escape sequence in a display name still reaches the terminal on the exact same line the new comment claims is safe. The asymmetry is also easy to reintroduce later, since the two cell paths in one function now have different sanitizing rules.

Suggested fix: make `sanitizeSearchCell` run `stripControl` first and drop its own `\r` replacement (`s = stripControl(s)` then the existing `\n` and `|` replacements). That gives one rule for every cell in both renderers, and closes the same gap in `SearchTable` at no extra cost; add a case to the existing sanitize test with an ESC in a meta cell to pin it.

*Reply inline to this comment.*

**Review by monit-reviewer — APPROVED** [review]
## Automated PR Review

**Reviewed commit:** `081c7d296553`
**Profile:** `claude-monit-reviewer` - **Posting as:** `monit-reviewer`

### Summary

| Reviewer | Findings |
|----------|----------|
| documentation:docs | 0 |
| go:implementation-tests | 0 |
| policies:conventions | 0 |
| architecture:solid-reviewer-agnostic | 1 |
| security:code-auditor | 0 |

<details>
<summary><strong>architecture:solid-reviewer-agnostic (1 finding)</strong></summary>

### Minor - `internal/output/output.go:181`

> U-L1: the control-character hardening added in this revision covers only the body, so the property `stripControl`'s comment states ("text another workspace member wrote cannot carry terminal escape sequences into the output") does not hold for the block's own meta line. The meta cells at line 181 go through `sanitizeSearchCell`, which drops `\r`, folds `\n` to a space and escapes `|`, but passes ESC, BEL and C1 characters straight to the terminal. In `search messages` the meta cells include `m.Channel.Name` and `m.Username`, both attacker-influenced display strings, so an escape sequence in a display name still reaches the terminal on the exact same line the new comment claims is safe. The asymmetry is also easy to reintroduce later, since the two cell paths in one function now have different sanitizing rules.
> 
> Suggested fix: make `sanitizeSearchCell` run `stripControl` first and drop its own `\r` replacement (`s = stripControl(s)` then the existing `\n` and `|` replacements). That gives one rule for every cell in both renderers, and closes the same gap in `SearchTable` at no extra cost; add a case to the existing sanitize test with an ESC in a meta cell to pin it.

</details>

### Reviewer Coverage

- `documentation:docs` — complete (constrained); inspected 2 assigned files (9 inspected across reviewers): `README.md`, `integration-tests.md`; skipped: none; constraints: none
- `go:implementation-tests` — complete (constrained); inspected 7 assigned files (9 inspected across reviewers): `internal/cmd/search/all.go`, `internal/cmd/search/files.go`, `internal/cmd/search/full_test.go`, `internal/cmd/search/messages.go`, `internal/cmd/search/render.go`, `internal/output/output.go`, `internal/output/output_test.go`; skipped: none; constraints: none
- `policies:conventions` — complete (constrained); inspected 5 assigned files (9 inspected across reviewers): `internal/cmd/search/all.go`, `internal/cmd/search/files.go`, `internal/cmd/search/full_test.go`, `internal/cmd/search/messages.go`, `internal/cmd/search/render.go`; skipped: none; constraints: Sibling repos open-cli-collective/cli-common and open-cli-collective/.github were not checked out locally, so shared command-surface and output-and-rendering conventions could only be checked against the repo-local breadcrumbs and summaries, not their full text.
- `architecture:solid-reviewer-agnostic` — complete (constrained); inspected 2 assigned files (9 inspected across reviewers): `internal/cmd/search/render.go`, `internal/output/output.go`; skipped: none; constraints: Both earlier inline threads on output.go (body contract wording, control characters) are marked settled in the dossier and were re-checked against 081c7d2 rather than re-reported. Scope limited to the two assigned files; the three command files, tests and docs were read only as context for the new selection point. Verification: `go test ./internal/output/...` and `CGO_ENABLED=0 go test ./internal/cmd/search/...` both pass. A cgo build failure in this workbench came from a harness temp/module-cache path, not the diff. Lint not run.
- `security:code-auditor` — complete (constrained); inspected 2 assigned files (9 inspected across reviewers): `internal/output/output.go`, `internal/output/output_test.go`; skipped: none; constraints: none

<details>
<summary>Inspected files (9)</summary>

- `README.md`
- `integration-tests.md`
- `internal/cmd/search/all.go`
- `internal/cmd/search/files.go`
- `internal/cmd/search/full_test.go`
- `internal/cmd/search/messages.go`
- `internal/cmd/search/render.go`
- `internal/output/output.go`
- `internal/output/output_test.go`

</details>

*0 PR discussion threads considered. 0 summarized; 0 resolved.*

---
<details>
<summary>Completed in 2m 04s | ~$2.03 (est.) | claude-sonnet-5, claude-opus-5 | cr 0.10.311</summary>

| Field | Value |
|---|---|
| Model | claude-sonnet-5, claude-opus-5 |
| Reviewers | documentation:docs, go:implementation-tests, policies:conventions, architecture:solid-reviewer-agnostic, security:code-auditor |
| Engine | claude_cli · claude-sonnet-5, claude-opus-5 |
| Reviewed by | cr · monit-reviewer |
| Duration | 2m 04s wall · 4m 37s compute |
| Cost | ~$2.03 (est.) |
| Pricing basis | anthropic-public-2026-09-02 |
| Tokens | 64 in / 10.2k out |

**Per-workstream usage**

- `documentation:docs` — claude-sonnet-5
  - In: 8
  - Out: 879
  - Cache read: 273.9k
  - Cache create: 52.0k
  - Cost: ~$0.27 (est.)
  - Duration: 29s
- `go:implementation-tests` — claude-sonnet-5
  - In: 18
  - Out: 2.5k
  - Cache read: 799.9k
  - Cache create: 66.1k
  - Cost: ~$0.45 (est.)
  - Duration: 1m 18s
- `policies:conventions` — claude-sonnet-5
  - In: 6
  - Out: 1.0k
  - Cache read: 208.4k
  - Cache create: 61.6k
  - Cost: ~$0.30 (est.)
  - Duration: 30s
- `architecture:solid-reviewer-agnostic` — claude-opus-5
  - In: 18
  - Out: 4.4k
  - Cache read: 851.4k
  - Cache create: 17.9k
  - Cost: ~$0.72 (est.)
  - Duration: 1m 40s
- `security:code-auditor` — claude-sonnet-5
  - In: 8
  - Out: 979
  - Cache read: 303.6k
  - Cache create: 33.9k
  - Cost: ~$0.21 (est.)
  - Duration: 29s
- `orchestrator-rollup` — claude-sonnet-5
  - In: 6
  - Out: 442
  - Cache read: 216.8k
  - Cache create: 11.0k
  - Cost: ~$0.09 (est.)
  - Duration: 9s

</details>

File: internal/output/output.go (hunk 3 above)
Comment (piekstra) [reply internal/output/output.go]: Fixed in e15b5d8: sanitizeSearchCell runs stripControl first, so every cell in SearchTable and the SearchBlocks meta line drops control characters the same way the body does. TestSearchCellsStripControlCharacters pins an ESC in a display name and a BEL in a table cell.

File: internal/cmd/search/render.go [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+// fullNoticeThreshold is the number of full-text results above which --full
+// prints a size notice. Full text is unbounded per result, so a page of them
+// can be long to read and costly for a caller that feeds the output to a
+// model.
+const fullNoticeThreshold = 10
+
+// noticeWriter receives the --full size notice; stderr keeps it out of the
+// results a caller parses.
+var noticeWriter io.Writer = os.Stderr
```
Comment (monit-reviewer) [inline internal/cmd/search/render.go]: U-D1/U-D2: `noticeWriter` is new mutable package-level state in a command package, and it is a second writer seam alongside the one the repo already designates. `internal/output` owns the rendering destination (`output.Writer`, swapped in tests the same way), while every other stderr warning in the repo writes to `os.Stderr` directly (`internal/cmd/messages/send.go:256`, `internal/cmd/initcmd/init.go:116`, `internal/config/relocate.go:265`). This revision introduces a third shape: a private global in `internal/cmd/search` that exists only so `TestFullNoticeAboveThreshold` can capture the notice. The next command that wants a stderr notice either copies the global into its own package or reaches for `os.Stderr` and is untestable, and tests that mutate two different writer globals have to keep both save/restore pairs in step.

Close call worth stating: mirroring `output.Writer` is defensible and the blast radius is one internal package, which is why this is minor rather than major. The cheap fix is to move the seam to where the repo already wires output: add `ErrWriter io.Writer = os.Stderr` to the `var` block in `internal/output/output.go` next to `Writer`, have `renderSearchRows` write the notice there, and drop the `io`/`os` imports from render.go. That keeps one composition root for both destinations and makes the seam reusable by the existing warning sites.

*Reply inline to this comment.*

**Review by monit-reviewer — APPROVED** [review]
## Automated PR Review

**Reviewed commit:** `23cb69b75e4f`
**Profile:** `claude-monit-reviewer` - **Posting as:** `monit-reviewer`

### Summary

| Reviewer | Findings |
|----------|----------|
| documentation:docs | 0 |
| go:implementation-tests | 0 |
| policies:conventions | 0 |
| architecture:solid-reviewer-agnostic | 1 |
| security:code-auditor | 0 |

<details>
<summary><strong>architecture:solid-reviewer-agnostic (1 finding)</strong></summary>

### Minor - `internal/cmd/search/render.go:23`

> U-D1/U-D2: `noticeWriter` is new mutable package-level state in a command package, and it is a second writer seam alongside the one the repo already designates. `internal/output` owns the rendering destination (`output.Writer`, swapped in tests the same way), while every other stderr warning in the repo writes to `os.Stderr` directly (`internal/cmd/messages/send.go:256`, `internal/cmd/initcmd/init.go:116`, `internal/config/relocate.go:265`). This revision introduces a third shape: a private global in `internal/cmd/search` that exists only so `TestFullNoticeAboveThreshold` can capture the notice. The next command that wants a stderr notice either copies the global into its own package or reaches for `os.Stderr` and is untestable, and tests that mutate two different writer globals have to keep both save/restore pairs in step.
> 
> Close call worth stating: mirroring `output.Writer` is defensible and the blast radius is one internal package, which is why this is minor rather than major. The cheap fix is to move the seam to where the repo already wires output: add `ErrWriter io.Writer = os.Stderr` to the `var` block in `internal/output/output.go` next to `Writer`, have `renderSearchRows` write the notice there, and drop the `io`/`os` imports from render.go. That keeps one composition root for both destinations and makes the seam reusable by the existing warning sites.

</details>

### Reviewer Coverage

- `documentation:docs` — complete (constrained); inspected 2 assigned files (9 inspected across reviewers): `README.md`, `integration-tests.md`; skipped: none; constraints: none
- `go:implementation-tests` — complete (constrained); inspected 7 assigned files (9 inspected across reviewers): `internal/cmd/search/all.go`, `internal/cmd/search/files.go`, `internal/cmd/search/full_test.go`, `internal/cmd/search/messages.go`, `internal/cmd/search/render.go`, `internal/output/output.go`, `internal/output/output_test.go`; skipped: none; constraints: none
- `policies:conventions` — complete (constrained); inspected 5 assigned files (9 inspected across reviewers): `internal/cmd/search/all.go`, `internal/cmd/search/files.go`, `internal/cmd/search/full_test.go`, `internal/cmd/search/messages.go`, `internal/cmd/search/render.go`; skipped: none; constraints: Sibling repos open-cli-collective/cli-common and open-cli-collective/.github were not checked out locally, so shared command-surface and output-and-rendering conventions could only be checked against the repo-local breadcrumbs and summaries, not their full text.
- `architecture:solid-reviewer-agnostic` — complete (constrained); inspected 2 assigned files (9 inspected across reviewers): `internal/cmd/search/render.go`, `internal/output/output.go`; skipped: none; constraints: Scope limited to the two assigned files; the command files, tests and docs were read only as context for the new selection point and the --full notice. The three inline threads on output.go are marked settled in the dossier and were re-checked against 23cb69b rather than re-reported. Verification: `go test ./internal/output/... ./internal/cmd/search/...` passes (CGO_ENABLED=0; a cgo build failure in this workbench came from a harness module-cache path, not the diff). Lint not run.
- `security:code-auditor` — complete (constrained); inspected 2 assigned files (9 inspected across reviewers): `internal/output/output.go`, `internal/output/output_test.go`; skipped: none; constraints: none

<details>
<summary>Inspected files (9)</summary>

- `README.md`
- `integration-tests.md`
- `internal/cmd/search/all.go`
- `internal/cmd/search/files.go`
- `internal/cmd/search/full_test.go`
- `internal/cmd/search/messages.go`
- `internal/cmd/search/render.go`
- `internal/output/output.go`
- `internal/output/output_test.go`

</details>

*0 PR discussion threads considered. 0 summarized; 0 resolved.*

---
<details>
<summary>Completed in 1m 43s | ~$1.76 (est.) | claude-sonnet-5, claude-opus-5 | cr 0.10.311</summary>

| Field | Value |
|---|---|
| Model | claude-sonnet-5, claude-opus-5 |
| Reviewers | documentation:docs, go:implementation-tests, policies:conventions, architecture:solid-reviewer-agnostic, security:code-auditor |
| Engine | claude_cli · claude-sonnet-5, claude-opus-5 |
| Reviewed by | cr · monit-reviewer |
| Duration | 1m 43s wall · 3m 27s compute |
| Cost | ~$1.76 (est.) |
| Pricing basis | anthropic-public-2026-09-02 |
| Tokens | 56 in / 9.6k out |

**Per-workstream usage**

- `documentation:docs` — claude-sonnet-5
  - In: 10
  - Out: 1.1k
  - Cache read: 448.0k
  - Cache create: 22.0k
  - Cost: ~$0.19 (est.)
  - Duration: 24s
- `go:implementation-tests` — claude-sonnet-5
  - In: 10
  - Out: 2.0k
  - Cache read: 465.8k
  - Cache create: 79.3k
  - Cost: ~$0.43 (est.)
  - Duration: 47s
- `policies:conventions` — claude-sonnet-5
  - In: 8
  - Out: 1.1k
  - Cache read: 390.4k
  - Cache create: 20.2k
  - Cost: ~$0.17 (est.)
  - Duration: 25s
- `architecture:solid-reviewer-agnostic` — claude-opus-5
  - In: 14
  - Out: 4.1k
  - Cache read: 779.4k
  - Cache create: 17.6k
  - Cost: ~$0.67 (est.)
  - Duration: 1m 20s
- `security:code-auditor` — claude-sonnet-5
  - In: 8
  - Out: 877
  - Cache read: 373.1k
  - Cache create: 30.7k
  - Cost: ~$0.21 (est.)
  - Duration: 18s
- `orchestrator-rollup` — claude-sonnet-5
  - In: 6
  - Out: 426
  - Cache read: 233.1k
  - Cache create: 10.5k
  - Cost: ~$0.09 (est.)
  - Duration: 10s

</details>

File: internal/cmd/search/render.go (hunk 4 above)
Comment (piekstra) [reply internal/cmd/search/render.go]: Fixed in 9ff52fc: added output.ErrWriter next to output.Writer, the notice writes there, and render.go keeps no writer global or io/os imports. The test swaps output.ErrWriter the same way other tests swap output.Writer.
